docs/2.8.0/docs/canton/usermanual/kms/external_key_storage_aws.rst

```diff
@@ -0,0 +1,199 @@
+   :end-before: user-manual-entry-end: ManualRegisterKmsKeys
+   :dedent:
+
+where `xyzKmsKeyId` is the AWS KMS identifier for a specific key (e.g. `KMS Key RN`). If we are using, for example,
+`AWS cross account keys <https://docs.aws.amazon.com/kms/latest/developerguide/key-policy-modifying-external-accounts.html>`_
+be aware that using the key id is not enough and we are required to register the key using its `ARN`.
+
+Here is a brief reminder of the initial keys that each node owns:
+
++-----------------------------------+------------+--------------------+---------------+--------------+-----------------+
+| **Keys**                          | **Domain** | **Domain-Manager** | **Sequencer** | **Mediator** | **Participant** |
++===================================+============+====================+===============+==============+=================+
+| `(Signing) Namespace Key`         | x          | x                  |               |              | x               |
++-----------------------------------+------------+--------------------+---------------+--------------+-----------------+
+| `Signing Key`                     | x          | x                  | x             | x            | x               |
++-----------------------------------+------------+--------------------+---------------+--------------+-----------------+
+| `(Asymmetric) Encryption Key`     |            |                    |               |              | x               |
++-----------------------------------+------------+--------------------+---------------+--------------+-----------------+
+
+Depending on the key purpose and the default signing and encryption schemes defined in Canton,
+you need to pre-generate the corresponding AWS KMS keys with the correct settings:
+
++-------------------+-----------------------------------------------------------------------+-----------------------------------------------------+
+| **Provider**      | **SIGNING**                                                           | **ENCRYPTION**                                      |
++===================+=======================================================================+=====================================================+
+| AWS               | - **Key Purpose:** `SIGN_VERIFY`                                      | - **Key Purpose:** `ENCRYPT_DECRYPT`                |
+|                   | - **Key Algorithms:** `ECC_NIST_P256` or `ECC_NIST_P384`              | - **Key Algorithm:** `RSA_2048`                     |
++-------------------+-----------------------------------------------------------------------+-----------------------------------------------------+
+
+Finally, using the previously registered keys, **we need to initialize our nodes**:
+
+- :ref:`initialize domain <manually-init-domain>`
+- :ref:`initialize participants <manually-init-participant>`
+
+Please be aware that for this specific use-case **we must use the keys that we registered and not generate new ones**.
```

Comment:
I'd put this before the links to the manual init doc. Because people will click on the link and possibly miss this and copy paste the script that generate new keys. So something like

```
Below are links to guides on how to manually initialize participant and domain nodes. In those guides, keys are generated using console commands such as:

val identityKey =
  participant.keys.secret.generate_signing_key(name = participant.name + "-namespace")

Make sure to replace those commands with the ones shown above with which we registered our existing keys instead of generating new ones.
```